runtimes/adapters/internal_agent.py

```
@@ -415,19 +415,55 @@ async def execute(self, spec: TaskSpec) -> TaskResult:
+                    await _e2b_session.call_tool("clone_repo", {
+                        "workspace_id": "default",
+                        "repo_url": clone_url,
+                        "branch": base_branch,
+                    })
+                    self._log.info(
+                        "E2B: cloned %s (branch=%s) into sandbox for task %s",
+                        repo_url, base_branch, spec.task_id,
                     )
-            except Exception as exc:  # pragma: no cover - best-effort
-                self._log.debug("E2B seed failed (best-effort): %s", exc)
+                except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)